lib/data/multipleBackendGateway.js

```diff
@@ -43,7 +43,7 @@ const multipleBackendGateway = {
         }
 
         return client.put(writeStream, size, keyContext,
-            reqUids, (err, key, eTag) => {
+            reqUids, (err, key, dataStoreVersionId) => {
```

Comment:
etag never needed?

Reply:
seems like it was left-over artifact when we were returning the eTag from aws, which we no longer do -- I checked with @dora-korpar to make sure